tests/onnx/test_onnx_v2.py

```diff
@@ -39,6 +46,38 @@ class OnnxUtilsTestCaseV2(TestCase):
     Cover all the utilities involved to export ONNX models
     """
 
+    @classmethod
+    def setUpClass(cls):
+        # Create local checkpoints - one time setup
+        test_model = "mrm8488/bert-tiny-finetuned-squadv2"
```

Comment:
Do the tests also work with `SMALL_MODEL_IDENTIFIER` from `testing_utils.py`? That checkpoint is preferred since it's maintained by the HF team and less likely to vanish unexpectedly :)

Reply:
Didn't realize that was an available variable to use! That works, done in [67416f2](https://github.com/huggingface/transformers/pull/18615/commits/67416f22094bb29b7bdcfe22854ee00ef4e1e112).